integ/wf_state_api_fail_test.go

```diff
@@ -82,6 +83,6 @@ func doTestStateApiFail(t *testing.T, backendType service.BackendType) {
 		WorkflowRunId:  startResp.GetWorkflowRunId(),
 		WorkflowStatus: iwfidl.FAILED,
 		ErrorType:      ptr.Any(iwfidl.STATE_API_FAIL_MAX_OUT_RETRY_ERROR_TYPE),
-		ErrorMessage:   iwfidl.PtrString("statusCode: 400, responseBody: {}, errMsg: 400 Bad Request  (%!s(*string=<nil>))"),
+		ErrorMessage:   iwfidl.PtrString("statusCode: 400, responseBody: {}, errMsg: 400 Bad Request"),
```

Comment:
@longquanzheng does this work for you before? I'm getting integ test failure due to this extra string